source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -435,15 +435,16 @@ namespace PlayFab
 
         // Call SetUrl
         {
-            const std::string requestUrl = GetUrl(requestTask);
+            CallRequestContainer* pfReqContainer = static_cast<CallRequestContainer*>(requestTask.requestContainer.get());
+            const std::string requestUrl = pfReqContainer->GetFullUrl();
 
             jmethodID methodId = jniEnv->GetMethodID(GetHelper().GetHttpRequestClass(), "setUrl", "(Ljava/lang/String;)Z");
             if (methodId == nullptr)
             {
                 return false;
             }
 
-            jstring urlJstr = jniEnv->NewStringUTF(requestUrl.c_str());
+            auto urlJstr = jniEnv->NewStringUTF(requestUrl.c_str());
```

Comment:
This doesn't meet the requirements for auto use. I don't know the return type of NewStringUTF to be a jstring.

Reply:
Ditto 1'st reply